packages/custom-webpack/src/verbose-logger.ts

```diff
@@ -0,0 +1,32 @@
+import { isString, get } from 'lodash';
+import { Configuration } from 'webpack';
+
+import { CustomWebpackBuilderVerboseConfig } from './custom-webpack-builder-config';
+
+export class VerboseLogger {
```

Comment:
I prefer switching to functions (in this case closure with options). 
Didn't have to to make the change project-wise but it is planned. So any new code is better to be written in functional paradigm rather than OOP.